core_tic_tac_toe/io.py

```diff
@@ -1,5 +1,6 @@
 class Io:
     def get_player_input(self):
+        self.print_message("Enter a value between 1-9")
```

Comment:
I think IO shouldn't really have to know about the game and how it works. I would recommend that you pass in the messaged as an argument. It's possible that you will want other human input such as their name.